addons/main/settings.sqf

```diff
@@ -50,6 +50,16 @@ private _curCat = LSTRING(Settings_MainCat);
     1
 ] call CBA_fnc_addSetting;
 
+// Toggles AI vehicle autonomous munition switching
+[
```

Comment:
do we want to do this like the other systems and use "disable" instead of "enable"? most of the other systems have settings like that